kong/templates/nginx_kong.lua

```diff
@@ -57,6 +56,14 @@ init_worker_by_lua_block {
     kong.init_worker()
 }
 
+upstream kong_upstream {
+    server 0.0.0.1;
+    balancer_by_lua_block {
+        kong.balancer()
+    }
+    keepalive 60;
```

Comment:
Note for polishing this PR: make this configurable. 
Also worth considering allowing users to add custom `upstream` blocks (managed by our template) to allow for different upstream connection pools maybe? With different keepalive settings.